Module_7.3/Q1.js

```diff
@@ -0,0 +1,45 @@
+// Write a function to check deep equality of two nested objects/arrays
+
+function deepEqual(obj1, obj2) {
+  if (typeof obj1 === 'object' && obj1 !== null &&
+      typeof obj2 === 'object' && obj2 !== null) {
+  
+      const keys1 = Object.keys(obj1);
```

Comment:
Have you checked for array first? 
This code would be executed for arrays as well & no use of comparing array keys